Dev/VideoChatClient/Input/AudioInput.cpp

```diff
@@ -0,0 +1,39 @@
+#include "AudioInput.hpp"
+
+
+AudioInput::AudioInput() :
+	IInput(PacketDataTypes::AudioDataType)
+{
+	const std::vector<std::string> available_devices = sf::SoundRecorder::getAvailableDevices();
+	for (const auto& available_device : available_devices)
+	{
+		std::cout << available_device << std::endl;
+	}
+	if (available_devices.empty() || !sf::SoundBufferRecorder::isAvailable())
+	{
+		throw VideoChatClientException(VideoChatClientStatus::AUDIO_INPUT_AUDIO_DEVICE_NOT_AVAILABLE);
+	}
+}
+
+void AudioInput::take_input()
+{
+	sf::SoundBufferRecorder recorder{};
+	recorder.start();
+	Sleep(SAMPLING_DURATION);
+	recorder.stop();
```

Comment:
Start and stop return value checks? Or they are void?

Reply:
Start is bool, stop void